### PR TITLE
test: manually clear and populate db and bucket for functional tests

### PR DESCRIPTION
**Pull request by Cooryd**
This will allow us to scale up the functional tests & make them more reliable.

### CODE REVIEW

File: test/functional/test-article.spec.ts [hunk 1]
```diff
@@ -13,33 +25,46 @@ describe('Get /article/id', () => {
...
 
   test('Can get article manifest', async () => {
+    await populateCollection('articles', [{...article, xml: '<article><body><fig id="fig1" position="float"><graphic xlink:href="elife-54296-fig1.tif" mimetype="image" mime-subtype="tiff"/></fig></body></article>'}]);
     return agent
       .get('/articles/54296/manifest')
       .expect('Content-Type', /json/)
-      .expect(200);
+      .expect(200)
+      .then(response => {
+        expect(response.body).toEqual({"articleId": "54296", "assets": [{"id": "fig1", "path": "http://localhost:8080/api/v1/articles/54296/assets/elife-54296-fig1.tif", "type": "image/jpeg"}], "path": "/api/v1/articles/54296/export", "type": ""});
```
Comment (NuclearRedeye) [inline test/functional/test-article.spec.ts:66]: I notice here that the resource is `.tif` but the mime type is `image/jpeg`. Is that expected?

**Review by NuclearRedeye — APPROVED** [review]
I've had a look through and it seems good to me. Noticed thought that there is a mismatch in mime type in the manifest which I think needs fixing, although that can be done in a follow up task as it seems unrelated to this change.

**Comment by NuclearRedeye** [comment]
Oh, and the failing test in the CI needs resolving :-)

File: test/functional/test-article.spec.ts (hunk 1 above)
Comment (Cooryd) [reply test/functional/test-article.spec.ts:67]: This seems to be hard coded to `"type": "image/jpeg"` in the article service `getManifest`. I'll raise an issue to make type detectable from the file.

**Pull request merged**